odoo/http.py

```diff
@@ -1798,6 +1799,7 @@ def _serve_db(self):
                     return self._serve_nodb()
             ir_http = self.registry['ir.http']
             self.env = odoo.api.Environment(cr, self.session.uid, self.session.context)
+            self.update_context(lang=get_lang(self.env).code)
```

Comment:
```suggestion
            self.update_context(lang=get_lang(self.env(user=odoo.SUPERUSER_ID)).code)
```

Reply:
Bad idea actually, `get_lang` will use the user to fallback on his company's lang. There is a chance that `SUPERUSER_ID` and `session.uid` are not in the same company.